pkg/proc/gdbserial/gdbserver.go

```diff
@@ -1329,7 +1337,7 @@ func (t *Thread) Blocked() bool {
 // inferior's thread.
 func (p *Process) loadGInstr() []byte {
 	var op []byte
-	switch p.bi.GOOS {
+	switch runtime.GOOS {
```

Comment:
Given what we are doing now this is fine however in principle one could use the gdbserial package to connect to a stub running on a different OS/architecture than the one where delve is running. We'll probably never do this but there's no reason to make this harder, either.

Reply:
Ack, agreed.